docker-compose.override.yml

```diff
@@ -4,13 +4,7 @@ services:
     command: python src/manage.py runserver 0.0.0.0:8000
     volumes:
       - .:/code
-      - /code/src/sous-chef
-      - /code/src/sous-chef/static
     expose:
       - "8000"
     ports:
```

Comment:
This exposes 0.0.0.0:8000 to outside the container, right? It's unclear to me if we need it, because we can access 0.0.0.0:80 just fine in development mode.